app/helpers/analytics_helper.rb

```diff
@@ -20,7 +20,7 @@ def track_analytics_event(event_category, event_action, options={})
       label_string = ",#{format_event_label(label, label_is_script)},#{value}"
     end
     event_args = "'#{event_category}','#{event_action}'#{label_string}"
-    "if (pageTracker) { pageTracker._trackEvent(#{event_args}) };"
+    "if (ga) { ga('send','event',#{event_args}) };"
```

Comment:
Just noting to myself that https://github.com/mysociety/alaveteli/pull/3160 is the PR for #3107.